File: Apps/Android/monitor-ui/src/main/kotlin/com/opencapture/monitorui/MonitorExposureReadout.kt
package com.opencapture.monitorui

/** Auto-exposure chip copy. EV stays the value; camera-chosen shutter stays in the caption. */
object MonitorExposureReadout {
    fun autoEvCaption(shutterDenom: Int): String =
        if (shutterDenom > 0) "EV 1/${shutterDenom}s" else "EV"
}

File: Sources/MonitorPresentation/MonitorExposureReadout.swift
import Foundation

/// Auto-exposure chip copy. EV stays the value; the camera-chosen shutter
/// remains readable in the caption.
public enum MonitorExposureReadout: Sendable {
    public static func autoEvCaption(shutterDenom: Int) -> String {
        shutterDenom > 0 ? "EV 1/\(shutterDenom)s" : "EV"
    }
}
